test/src/main/java/com/redhat/lightblue/test/Assert.java

```diff
@@ -0,0 +1,70 @@
+/*
+ Copyright 2015 Red Hat, Inc. and/or its affiliates.
+
+ This file is part of lightblue.
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.redhat.lightblue.test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.runners.model.MultipleFailureException;
+
+import com.redhat.lightblue.DataError;
+import com.redhat.lightblue.Response;
+import com.redhat.lightblue.util.Error;
+
+public final class Assert {
+
+    /**
+     * Asserts no Errors are on a {@link Response} from lightblue.
+     * @param response - {@link Response} to check.
+     * @throws MultipleFailureException
+     */
+    public static void assertNoErrors(Response response) throws MultipleFailureException{
+        List<Throwable> errors = new ArrayList<Throwable>();
+        for(Error error : response.getErrors()){
+            Exception e = new Exception(error.getMessage(), error);
+            e.printStackTrace();
```

Comment:
Two suggestions:
* if printing to standard out, pass a boolean argument to control if the stack trace is printed or not
* consider using a logger instead, which could be configured to write to a file or controlled with logging level.